scripts/division_io.py

```diff
@@ -223,13 +223,66 @@ def export_division_data(verbose: bool = False):
     dest_path.mkdir(parents=True, exist_ok=True)
 
     db_connection = get_twfy_db_connection()
+
+    # Create an export of the divisions data
+    # Join with hansard table to get the gid of the debate as well
+    # This is broadly the same as the old public whip dump
+    divisions_query = """
+    SELECT
+        division_id,
+        house as chamber,
+        CASE
+            WHEN division_id like '%-cy%' THEN 'cy' ELSE 'en'
+        END as language,
+        divisions.gid as source_gid,
+        hansard_debate.gid as debate_gid,
+        division_title,
+        yes_text,
+        no_text,
+        division_date,
+        division_number,
+        yes_total,
+        no_total,
+        absent_total,
+        both_total,
+        majority_vote,
+        lastupdate,
+        title_priority
+    FROM divisions
+    JOIN hansard
+        USING (gid)
+    JOIN hansard as hansard_debate
+        on hansard_debate.epobject_id = hansard.subsection_id
+    """
+
+    # Export of all voting information
+    # But we also want the membership_id of the person
+    # at the time of the vote
+    # to match old public whip dump format
+    # makes it easier to do simple joins to add party, etc
+    votes_query = """
+    SELECT
+        persondivisionvotes.person_id as person_id,
+        member.member_id as membership_id,
+        persondivisionvotes.division_id as division_id,
+        vote,
+        proxy
+    FROM
+        persondivisionvotes
+    JOIN divisions
+        ON persondivisionvotes.division_id = divisions.division_id
+    JOIN member
+        ON persondivisionvotes.person_id = member.person_id
+        AND divisions.division_date BETWEEN member.entered_house AND member.left_house;
```

Comment:
This will have an issue if someone is in two houses at once, that both have divisions, won't it?

Reply:
Ah right - it needs to also have a `divisions.house = member.house` I think? I'll give that a go at some point and check it outputs correctly.

Reply:
Ok, fix in a new commit which I'll squish. divisions.house is string and member.house is int so it needs the conversion in the query.  Adding this slightly decreased the number of divisions being exported as you'd hope.